bibliopixel/drivers/spi_driver_base.py

```diff
@@ -1,75 +1,155 @@
-import os, sys, time
+import fcntl
```

Comment:
Thx, haven't know this isn't supported on windows,
Committed a patch that do a try catch ImportError like that for`spidev`.

Is in my opinion much simpler than check for the OS, if this works